README.md

```diff
@@ -1,64 +1,67 @@
-# External DNS
+# ExternalDNS
 [![Build Status](https://travis-ci.org/kubernetes-incubator/external-dns.svg?branch=master)](https://travis-ci.org/kubernetes-incubator/external-dns)
 [![Coverage Status](https://coveralls.io/repos/github/kubernetes-incubator/external-dns/badge.svg?branch=master)](https://coveralls.io/github/kubernetes-incubator/external-dns?branch=master)
 [![GitHub release](https://img.shields.io/github/release/kubernetes-incubator/external-dns.svg)](https://github.com/kubernetes-incubator/external-dns/releases)
 
-ExternalDNS synchronizes exposed Services and Ingresses with cloud DNS providers.
+ExternalDNS synchronizes exposed Services and Kubernetes Ingresses with cloud DNS providers.
```

Comment:
This should be "exposed Kubernetes Services and Ingresses" then. I'm also thinking about dropping the "cloud" from DNS providers as it's not limited to them.